docs/lesiaturuta/PokemonProject/loader/loader.css

```diff
@@ -0,0 +1,32 @@
+@import url("https://fonts.googleapis.com/css?family=Roboto:100");
```

Comment:
why do import the font here? Please move to `style.css`

Reply:
fixed